### ISSUE TITLE
160939580 List of Counties "Open" marker (Eric D)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1274]
Issue reported by **REGManager** at 2020-02-06 17:26:20 UTC
Time: 2020-02-06T17:23:54+00:00
Session ID: 667e08bb350a7c7990e4089b7477f1bb
Problem Page URL: [/counties?locale=en](/counties?locale=en)
Previous Page URL: [https://www.freereg.org.uk/counties/531a1999231040725c000055/edit?locale=en](https://www.freereg.org.uk/counties/531a1999231040725c000055/edit?locale=en)
Reported Issue:
What does the A and U stand for in this field and  how is it edited?    

![Screenshot](https://www.freereg.org.uk)

**Comment by Captainkirkdawson** [comment]
Sorry do not understand; what display? Found it. Beyond me I have no idea

**Comment by edickens** [comment]
County Coordinators Action.
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/2049d51b-c3dc-4469-9734-7363a87e46a9)
Column headed "Open"
What does it mean and how is it edited?

**Comment by Captainkirkdawson** [comment]
@edickens you asked "What does it mean " and  my reply is "I do not know" It is not currently editable.

**Comment by edickens** [comment]
The report must have got it from somewhere and someone has added that field and programmed it into the report !!!  Or is it the result of a calculation, such as to say "Active".  Or is it a field in the CCs UserID to do with the agreement?
Have just looked at the source:-
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/00bab95d-01aa-41a3-b825-e06cd6db5608) 
But not sure what is the significance of "Open"?

**Comment by Captainkirkdawson** [comment]
As you see from your post the column represents the Transcriber Agreement Accepted status of the coordinator. 
Clearly was added so you know if the coordinator had themselves accepted the Transcriber agreement. So what is there is their status A: accepted and U: Unknown. You cannot edit; it must be done by the individual coordinators themselves.
The same field is there for the Syndicate Coordinators 
Bottom line I guess is it lets management know if their coordinators have themselves accepted the agreement
So they were added whenever there was the push on Transcriber Agreement.
Guess I must have done the code but have no recollection of so doing. Age does rob one of memories especially if they have a low level of interest.

**Comment by edickens** [comment]
TY.
Can be closed.
